source/include/backoff_algorithm.h

```diff
@@ -36,6 +36,12 @@
 /* Standard include. */
 #include <stdint.h>
 
+#ifdef DISABLE_LOGGING
```

Comment:
This should be named DISABLE_ASSERTS since it disables asserts and logging.

Also, this cannot be in the h file as it will affect all downstream object files, including customer code that includes our library. This must be done in the c file.

Reply:
That's a fair point, was a bit of carry over from when I had first been using it to remove logging. Thanks for pointing it out. I'll move it to the c file as well.